js/modules/k6/experimental/tracing/client_test.go

```diff
@@ -144,6 +149,105 @@ func TestClientInstrumentArguments(t *testing.T) {
 	})
 }
 
+func TestClientInstrumentedCall(t *testing.T) {
+	t.Parallel()
+
+	testCase := newTestCase(t)
+	testCase.testSetup.MoveToVUContext(&lib.State{
+		Tags: lib.NewVUStateTags(&metrics.TagSet{}),
+	})
+	testCase.client.propagator = NewW3CPropagator(NewAlwaysOnSampler())
+
+	callFn := func(args ...goja.Value) error {
+		gotMetadataTraceID, gotTraceIDKey := testCase.client.vu.State().Tags.GetCurrentValues().Metadata["trace_id"]
+		assert.True(t, gotTraceIDKey)
+		assert.NotEmpty(t, gotMetadataTraceID)
```

Comment:
You went through the trouble of having this to be deterministic by setting a specific `randSource` and ... then you didn't test it is the specific value we expect :(